thorlcr/slave/thslavemain.cpp

```diff
@@ -123,7 +118,11 @@ static bool RegisterSelf(SocketEndpoint &masterEp)
             assertex(mySlaveNum == configSlaveNum);
 
         globals.setown(createPTree(msg));
-        mergeCmdParams(globals); // cmd line
+
+        /* NB: preserve command line option overrides
+         * Not sure if any cmdline options are actually needed by this stage..
+         */
+        loadArgsIntoConfiguration(globals, cmdArgs);
```

Comment:
All components (including client tools I suspect) need to call loadConfiguration.  If the slave calls it, I don't think this code is needed.

Reply:
slaves do not have a configuration file, the master serializes the configuration from the master.
They do however have at least one command line param --master=, and in base metal case a few others.

@ghalliday  Is loadConfiguration appropriate without a configuration file?